.github/scripts/toolchain/build-mingw-winpthreads.sh

```diff
@@ -16,19 +16,12 @@ if [[ "$RUN_CONFIG" = 1 ]] || [[ ! -f "$MINGW_BUILD_PATH/Makefile" ]]; then
                 --enable-debug"
         fi
 
-        case "$ARCH" in
-            aarch64)
-                CFLAGS="$CFLAGS \
-                    -mno-outline-atomics"
-            ;;
-        esac
-
         $SOURCE_PATH/mingw/mingw-w64-libraries/winpthreads/configure \
             --prefix=$TOOLCHAIN_PATH/$TARGET \
             --build=$BUILD \
             --host=$TARGET \
             --enable-static \
-            --enable-shared \
+            --disable-shared \
```

Comment:
Can you elaborate please why you had to disable building shared version of `winpthreads`?

Reply:
I am glad you asked as this is quite important question: The reason is that when the stage 1 GCC is being built, it needs to be built with static `libgcc` only as stage1 compiler is not able to link binaries. Then the runtime, which `winpthreads` is part of, needs to be built but shared `libgcc` is not available yet so it needs to be built statically as well. Then the stage 2 GCC is built with all those dependencies with both static and shared `libgcc` and then `winpthread` is re-built again both static and share in `build-mingw.sh` step.

Reply:
Thank you, for shedding some light on it. Resolving the thread, so it's not blocking merging